sdk/test/test.go

```diff
@@ -362,19 +362,58 @@ func (s *Server) handleBundles(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// Prepare a mapping to store bundle data
+	data := map[string]interface{}{}
+
+	// Prepare a manifest for use if a .manifest file exists.
+	var manifest bundle.Manifest
+
 	// Prepare the modules to include in the bundle. Sort them so bundles are deterministic.
 	modules := make([]bundle.ModuleFile, 0, len(b))
 	for url, str := range b {
-		module, err := ast.ParseModule(url, str)
-		if err != nil {
+		switch {
+		case url == ".manifest":
+			err := json.Unmarshal([]byte(str), &manifest)
+			if err != nil {
+				w.WriteHeader(http.StatusInternalServerError)
+				fmt.Fprintf(w, "error unmarshaling .manifest file: %v", err)
+				return
+			}
+		case strings.HasSuffix(url, ".rego"):
+			module, err := ast.ParseModule(url, str)
+			if err != nil {
+				w.WriteHeader(http.StatusInternalServerError)
+				_, _ = w.Write([]byte(err.Error()))
+				return
+			}
+			modules = append(modules, bundle.ModuleFile{
+				URL:    url,
+				Parsed: module,
+			})
+		case strings.HasSuffix(url, ".json"):
+			if strings.Contains(url, "/") {
+				w.WriteHeader(http.StatusInternalServerError)
+				fmt.Fprintf(w, "nested data documents are not implemented in the dummy server: %s", url)
```

Comment:
This felt like a pragmatic option here since it wasn't what I was here to do. It seemed to make sense to support the loading of a basic data.json as part of a bundle as a drive by bonus, however, it didn't seem to make sense to implement a feature complete merging of data.jsons from nested paths in the bundle.

Happy to tear this out and circle back to it when we need it (I have added a test for the use of bundle data too).